docs/developer/app-store/apps/emails-and-messages/overview.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Overview
+sidebar_position: 1
+---
+
+# Introduction
+
+Emails and messages (EAM) application is responsible for sending emails and messages to the customers. It is connected to the Saleor API via webhooks, which are used to notify the application about the events. Delivery of the messages is done by the external providers, which are configured in the application.
+
+## Suported providers
+
+Currently, we are supporting:
+
+- [Sendgrid](sendgrid) - Create templates in the Sengrid visual editor and send them to your customers
+- [SMTP](smtp) - Use your own SMTP server to send emails. Email templates are created using MJML and handlebars.
+
+## Supported events
+
+The application is listening to the following events:
```

Comment:
**question:**

What type of events are these? Because I don't see them on the webhook events list.

Reply:
The list was copied from old plugin.

Reorganized this section to contain more info on where those events come from and added links for non-trivial topics like activating the account. WDYT?